Source/Extensions/DictionaryExtensions.swift

```diff
@@ -8,6 +8,17 @@
 
 import Foundation
 
+// MARK: - Properties
+public extension Dictionary {
+    
+    /// SwifterSwift: Return every key of the dictionary as a tuple array. There's no order guaranteed.
+    ///
+    /// - Returns: key/value of dictionary as a tuple array.
+    public var tuples : [(key: Key, value: Value)] {
```

Comment:
We really shouldn't use computed properties for functionality with complexity > O(1)